tools/model_tools/src/openvino/model_zoo/_common.py

```diff
@@ -98,6 +99,31 @@ def end_session(self, category): pass
         def force_shutdown(self, timeout): pass
 
 
+try:
+    from model_api.adapters import create_core, OpenvinoAdapter
+    from model_api.models import Model, Classification
+    from model_api.pipelines import get_user_config
+except ImportError:
```

Comment:
`model_api` becomes a part of openvino-dev just like model tools. If model_tools exists, `model_api` is also available. I think you can remove wrapping try except

Reply:
as far as I know model_api becomes part of openvino-dev at 2022.2 but not in 2022.1

cc @slyubimt is that correct?

Reply:
model_api is not targeted for 2022.1.

Reply:
ok, disregard my comment, this PR is targeted for 2022.2, just mistakenly thought we are some in other PR

Reply:
Updated imports